### PR TITLE
Adding RSV

### PR DESCRIPTION
**Pull request by RSVpool**

### CODE REVIEW

File: registry.json [hunk 1]
```diff
@@ -1,4 +1,19 @@
...
+      "social":{
+         "facebook":"",
+         "twitter":"@RSVNODE",
+         "website":"https://reservoir.network",
+         "reddit":"",
+         "instagram":"",
+         "youtube":""
+      },
+      "telegram":"@reservoirNode",
+      "operatorName":"contact",
```
Comment (lophre) [inline registry.json:14]: If you don't provide your Real name, please leave operatorName empty.

File: registry.json [hunk 2]
```diff
@@ -1,4 +1,19 @@
 [
+   {
+      "ticker":"RSV",
+      "poolId":"pool1r0sxz0hygfc0266lhdclve6j3cwcwlth6dya6uey65yxjnwm7qc",
```
Comment (lophre) [inline registry.json:4]: Your pool id is : 1be0613ee44270f56b5fbb71f667528e1d877d77d349dd7324d50869
Please update your PR.

**Review by lophre — CHANGES_REQUESTED** [review]
Please look at the comments and update...

File: registry.json (hunk 1 above)
Comment (RSVpool) [reply registry.json]: Hello, I'm new to github. Hope I didnt screw this up. thanks for understanding

**Comment by RSVpool** [comment]
looks like I did do it wrong, I apologize.

**Pull request closed without merging**